Initial checkin for Azure Managed Apps sample (#1)
Azure-Samples/communication-services-azure-managed-apps: SECURITY.md

<!-- BEGIN MICROSOFT SECURITY.MD V0.0.8 BLOCK -->

## Security

Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [our GitHub organizations](https://opensource.microsoft.com/).

> If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://aka.ms/opensource/security/definition), please report it to us as described below.

Reviewer: ```suggestion
If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://aka.ms/opensource/security/definition), report it as follows.
```

## Reporting Security Issues

> **Please do not report security vulnerabilities through public GitHub issues.**

Reviewer: ```suggestion
**Do not report security vulnerabilities through public GitHub issues.**
```

> Instead, please report them to the Microsoft Security Response Center (MSRC) at [https://msrc.microsoft.com/create-report](https://aka.ms/opensource/security/create-report).

Reviewer: ```suggestion
Instead, report security issues to the Microsoft Security Response Center (MSRC) at [https://msrc.microsoft.com/create-report](https://aka.ms/opensource/security/create-report).
```

> If you prefer to submit without logging in, send email to [secure@microsoft.com](mailto:secure@microsoft.com).  If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://aka.ms/opensource/security/pgpkey).

Reviewer: ```suggestion
If you prefer to submit without logging in, send email to [secure@microsoft.com](mailto:secure@microsoft.com). If possible, encrypt your message with our PGP key. Download it from the [Microsoft Security Response Center PGP Key page](https://aka.ms/opensource/security/pgpkey).
```

> You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Additional information can be found at [microsoft.com/msrc](https://aka.ms/opensource/security/msrc). 

Reviewer: ```suggestion
You should receive a response within 24 hours. If for some reason you do not, follow up via email to ensure we received your original message. For more information, see [microsoft.com/msrc](https://aka.ms/opensource/security/msrc). 
```

> Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:

Reviewer: ```suggestion
Include as much of the following requested information as you can to help us better understand the nature and scope of the possible issue:
```

>   * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)

Reviewer: ```suggestion
  * Type of issue (such as buffer overflow, SQL injection, cross-site scripting, and so on)
```

  * Full paths of source file(s) related to the manifestation of the issue
  * The location of the affected source code (tag/branch/commit or direct URL)
  * Any special configuration required to reproduce the issue
  * Step-by-step instructions to reproduce the issue
  * Proof-of-concept or exploit code (if possible)
  * Impact of the issue, including how an attacker might exploit the issue

> This information will help us triage your report more quickly.

Reviewer: ```suggestion
This information helps us quickly triage your report.
```

> If you are reporting for a bug bounty, more complete reports can contribute to a higher bounty award. Please visit our [Microsoft Bug Bounty Program](https://aka.ms/opensource/security/bounty) page for more details about our active programs.

Reviewer: ```suggestion
If you are reporting for a bug bounty, more complete reports can contribute to a higher bounty award. See our [Microsoft Bug Bounty Program](https://aka.ms/opensource/security/bounty) page for more details about our active programs.
```
